test/e2e/runner/load.go

```diff
@@ -27,73 +25,101 @@ func Load(ctx context.Context, testnet *e2e.Testnet, multiplier int) error {
 	if concurrency == 0 {
 		concurrency = 1
 	}
-	initialTimeout := 1 * time.Minute
-	stallTimeout := 30 * time.Second
 
 	chTx := make(chan types.Tx)
-	chSuccess := make(chan types.Tx)
+	chSuccess := make(chan int)
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
 	// Spawn job generator and processors.
 	logger.Info(fmt.Sprintf("Starting transaction load (%v workers)...", concurrency))
 	started := time.Now()
 
-	go loadGenerate(ctx, chTx, multiplier, testnet.TxSize)
+	go loadGenerate(ctx, chTx, testnet.TxSize)
 
 	for w := 0; w < concurrency; w++ {
 		go loadProcess(ctx, testnet, chTx, chSuccess)
 	}
 
-	// Monitor successful transactions, and abort on stalls.
+	// Montior transaction to ensure load propagates to the network
+	//
+	// note: previous versions of this loop ensured that the load
+	// didn't stall for more than 30 seconds, but checking for
+	// stalls here just aborts the load generator earlier, and can
+	// easily reflect backpressure in the test harness. What we
+	// should care about more than stalls is, 80th or 95th
+	// percentile latency, and this is not feasible to track here
+	// right now.
```

Comment:
I appreciate this comment. I suggest not anchoring the wording of the comment on the previous behaviour, though:

```go
// This loop doesn't check or time out for stalls, since a stall here just
// aborts the load generator sooner and could obscure backpressure
// from the test harness. Ideally we should monitor latency as a guide
// for when to give up, but we don't have a good way to track that yet.
```

or words to that effect.